### PR TITLE
8303408 : [AIX] Broken jdk8u build after JDK-8266391

### PR DESCRIPTION
**Pull request by deepa181**
The AIX build fails after [JDK-8266391](https://bugs.openjdk.org/browse/JDK-8266391) was backported to jdk8. So, We added null returning SystemMetrics class an aix platform specific class that fixes the build error.

Reported Issue : [JDK-8303408](https://bugs.openjdk.org/browse/JDK-8303408)

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8303408](https://bugs.openjdk.org/browse/JDK-8303408): [AIX] Broken jdk8u build after JDK-8266391

### Reviewers
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)
 * [Severin Gehwolf](https://openjdk.org/census#sgehwolf) (@jerboaa - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk8u pull/33/head:pull/33` \
`$ git checkout pull/33`

Update a local copy of the PR: \
`$ git checkout pull/33` \
`$ git pull https://git.openjdk.org/jdk8u pull/33/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 33`

View PR using the GUI difftool: \
`$ git pr show -t 33`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk8u/pull/33.diff">https://git.openjdk.org/jdk8u/pull/33.diff</a>

</details>

### CODE REVIEW

**Review by phohensee — APPROVED** [review]
Lgtm.

**Comment by phohensee** [comment]
Tagged the JBS issue.

**Comment by deepa181** [comment]
> Lgtm.

Thank you so much @phohensee for review.

**Comment by deepa181** [comment]
> Tagged the JBS issue.

Okay. Thanks @phohensee .

**Comment by deepa181** [comment]
I want more approvals for this fix before it gets Merge. Thanks .

**Review by jerboaa — APPROVED** [review]
Looks fine. Thanks for the fix!

**Comment by jerboaa** [comment]
The fix is fine. `Metrics` is a linux only (cgroups) feature. So returning null is the right thing to do on AIX.

**Comment by deepa181** [comment]
> The fix is fine. `Metrics` is a linux only (cgroups) feature. So returning null is the right thing to do on AIX.

Thanks @jerboaa  for review the code.

**Pull request closed without merging**
